addons/storyshots/storyshots-core/src/api/index.js

```diff
@@ -41,28 +35,51 @@ function testStorySnapshots(options = {}) {
     integrityOptions,
     snapshotSerializers,
   } = ensureOptionsDefaults(options);
-
   const testMethodParams = {
     renderTree,
     renderShallowTree,
     stories2snapsConverter,
   };
 
-  callTestMethodGlobals(testMethod);
+  const data = storybook
+    .raw()
+    .filter(({ name }) => (storyNameRegex ? name.match(storyNameRegex) : true))
+    .filter(({ kind }) => (storyKindRegex ? kind.match(storyKindRegex) : true))
+    .reduce((acc, item) => {
+      const { kind, story: render, parameters = {} } = item;
+      const existing = acc.find(i => i.kind === kind);
+      const { fileName } = item.parameters;
 
-  snapshotsTests({
-    groups: storiesGroups,
-    asyncJest,
-    suite,
-    framework,
-    storyKindRegex,
-    storyNameRegex,
-    testMethod,
-    testMethodParams,
-    snapshotSerializers,
-  });
+      if (parameters.storyshots !== false) {
```

Comment:
This should be `parameters.storyshots && ! parameters.storyshots.disabled`.

We decided at some point to not set `parameters.X = false` to mean disabled, instead `parameters.X = { disabled: true }`. If I recall right it was you that advocated for that @ndelangen  ;)

Reply:
/shrug